flink-cdc-connect/flink-cdc-pipeline-connectors/flink-connector-test-cdc/src/main/java/com/ververica/cdc/connectors/test/source/ValuesDataSourceOptions.java

```diff
@@ -0,0 +1,36 @@
+/*
+ * Copyright 2023 Ververica Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package com.ververica.cdc.connectors.test.source;
+
+import com.ververica.cdc.common.configuration.ConfigOption;
+import com.ververica.cdc.common.configuration.ConfigOptions;
+
+/** Configurations for {@link ValuesDataSource}. */
+public class ValuesDataSourceOptions {
+
+    public static final ConfigOption<Integer> TEST_CASE_INDEX =
+            ConfigOptions.key("testcase.index")
+                    .intType()
+                    .defaultValue(1)
+                    .withDescription("Index of test case.");
```

Comment:
What's the meaning ` TEST CASE`?  it confuse me a lot, do you mean something like `input records number`? you can give a better name for this purpose

Reply:
We may create several test cases for different scenario in [SourceEventBuilder](https://github.com/ververica/flink-cdc-connectors/pull/2675/files#diff-eae21b8b176e9d9562ab8f602ed401aa53e1891a3c95ca1e1a660b4202afb738), testcase.index is used to represent a case of the current selection.